VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -58,12 +58,17 @@ public override async Task OnConnectedAsync()
 
         private async Task AddUserToConferenceGroups(bool isAdmin)
         {
-            var conferenceIds = await GetConferenceIds(isAdmin);
+           var conferenceIds = await GetConferenceIds(isAdmin);
             var tasks = conferenceIds.Select(c => Groups.AddToGroupAsync(Context.ConnectionId, c.ToString())).ToArray();
 
             await Task.WhenAll(tasks);
         }
 
+        public async Task AddToGroup(string conferenceId)
+        {
+            await Groups.AddToGroupAsync(Context.ConnectionId, conferenceId);
+        }
```

Comment:
I have a slight concern over this being unrestricted. I'm not sure if this is important though. I'm going to take a quick look at if we can do things another way but if we stick with this could we restrict this to VHOs?

Reply:
I think so there is a function called IsSenderAdmin that might be something I can use, I am not sure i didnt think about that